### ISSUE TITLE
sera destro can't friendly fire with lazer

### ISSUE THREAD

**Issue by shalkya** [FAForever/fa#2122]
http://content.faforever.com/faf/vault/replay_vault/replay.php?id=6538612
log : https://gist.github.com/shalkya/9292293d6853713b0ae7d97a57879742

only torpedo does friendly fire

**Comment by CookieNoob** [comment]
https://github.com/FAForever/fa/blob/develop/units/XSS0201/XSS0201_unit.bp

all weapons of the destroyer have `CollideFriendly = false`.

**Comment by shalkya** [comment]
yes jagged told me afterward that this belong to balance discussion.
destro/BC wouldn't change because they would do too much friendly fire otherwise.
for example T2 PD does have friendly fire because it can fire higher than most land unit, so no need to remove friendly fire here.

But for CZAR/GC i still see no point removing friendly fire on them.

BTW to answer you cookie, it's weird that it's `CollideFriendly = false` for all weapon, when torpedo does friendly fire. (But not really important)
